tests/cleo_tests/compile.bat

```diff
@@ -0,0 +1,12 @@
+@REM Compile all .txt files in the current directory to .s files using Sanny Builder 4
```

Comment:
name the file `.Compile_All.bat`

Reply:
```
@REM Compile all .txt files in the current directory to .s files using Sanny Builder 4
@REM Usage: set SANNY="path\to\sanny.exe" && compile.bat

@Echo off

Setlocal enabledelayedexpansion
For /r %%i In (*.txt) Do (
    Set p=%%i
    Echo Compiling !p:%__CD__%=!

    If Exist "%%~dpi%%~ni.s" (Del /q "%%~dpi%%~ni.s")
    %SANNY% --compile "%%i" "%%~dpi%%~ni.s" --no-splash --mode sa_sbl
)

Echo Done.
Pause
```

Reply:
updated. I delete all .s files before recompiling test suite. in this case if a source file was deleted, .s file will be deleted too.

Reply:
Right!
I think the
```
Set p=%%i 
Echo Compiling !p:%__CD__%=!
```
part is worth transplanting into your version

Reply:
"Echo Compiling !p:%__CD__%=!" does not work to me in cmd.exe on Windows 10